explorer/client/src/utils/timeUtils.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { IS_STATIC_ENV } from './envUtil';
+
+const stdToN = (original: number, length: number) =>
+    String(original).padStart(length, '0');
+
+export const convertNumberToDate = (epochMilliSecs: number | null): string => {
+    if (!epochMilliSecs) return 'Not Available';
+
+    const date = new Date(epochMilliSecs);
+
+    const MONTHS = [
+        'Jan',
+        'Feb',
+        'Mar',
+        'Apr',
+        'May',
+        'Jun',
+        'Jul',
+        'Aug',
+        'Sep',
+        'Oct',
+        'Nov',
+        'Dec',
+    ];
+
+    return `${stdToN(date.getUTCDate(), 2)} ${
+        MONTHS[date.getUTCMonth()]
+    } ${date.getUTCFullYear()} ${stdToN(date.getUTCHours(), 2)}:${stdToN(
+        date.getUTCMinutes(),
+        2
+    )}:${stdToN(date.getUTCSeconds(), 2)} UTC`;
+};
+
+export const timeAgo = (
```

Comment:
can we switch this out for a library like [Luxon](https://github.com/moment/luxon) ?  
this works ok for our usual time format, but using a library means we benefit from handling locale differences & extensive testing.

Reply:
it's not for more advanced functionality that i propose using a library for this - it's for things that mostly vary by user - time zone and locale / number formatting.